src/hotspot/cpu/riscv/c1_CodeStubs_riscv.cpp

```diff
@@ -93,8 +93,8 @@ void RangeCheckStub::emit_code(LIR_Assembler* ce) {
     stub_id = C1StubId::throw_range_check_failed_id;
   }
   // t0 and t1 are used as args in generate_exception_throw，
-  // so use ra as the tmp register for rt_call.
-  __ rt_call(Runtime1::entry_for(stub_id), ra);
+  // so use x18 as the tmp register for rt_call.
+  __ rt_call(Runtime1::entry_for(stub_id), x18);
```

Comment:
Can we use `x9` instead here? It's the first callee-save register in numbering.

Reply:
Fixed